cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Azure.store.tsx

```diff
@@ -17,6 +17,7 @@ const initialState = {
         AZURE_ENVIRONMENT: AzureCloud.PUBLIC,
         REGION: '',
         SSH_PUBLIC_KEY: '',
+        IMAGE_INFO: {},
```

Comment:
I would rename this to OS_IMAGE for clarity and simplicity

Reply:
Done